ConsoleApp/School/lesson.cs

```diff
@@ -0,0 +1,9 @@
+namespace ConsoleApp.School;
+ 
+public enum lesson
+{
+    english,
```

Comment:
code convention violated here
```suggestion
    English,
```